app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1396,6 +1399,8 @@ class BrowserTabFragment :
         private const val TRACKERS_INI_DELAY = 500L
         private const val TRACKERS_SECONDARY_DELAY = 200L
 
+        private const val DEFAULT_CIRCLE_TARGET_TIMES_1_5 = 96
```

Comment:
isn't this default target zone `48dp` times 2?

Reply:
It actually is `DEFAULT_CIRCLE_TARGET` times 1.5 (64 * 1.5). I checked the code `setDistanceToTriggerSync` sets `mTotalDragDistance`. By default it does:

```
mSpinnerOffsetEnd = (int) (DEFAULT_CIRCLE_TARGET * metrics.density);
mTotalDragDistance = mSpinnerOffsetEnd;
```